### PR TITLE
Update the vscode GitHub Pull Request plugin

### PR DESCRIPTION
**Pull request by vinokurig**
Signed-off-by: Igor Vinokur <ivinokur@redhat.com>

### What does this PR do?

Update the vscode GitHub Pull Request plugin with the latest version

depends on https://github.com/eclipse/che-theia/pull/841

### CODE REVIEW

**Review by ericwill — CHANGES_REQUESTED** [review]
Please add a new folder with version `0.20.0` and new `meta.yaml`, instead of editing the existing one. Please also update the [vscode-extensions.json entry](https://github.com/eclipse/che-plugin-registry/blob/master/vscode-extensions.json#L195) for this plugin to latest upstream revision (looks like it's `0.20.0`).

**Comment by vinokurig** [comment]
@ericwill 
> Please add a new folder with version 0.20.0 and new meta.yaml, instead of editing the existing one. Please also update the vscode-extensions.json entry for this plugin to latest upstream revision (looks like it's 0.20.0).

Done

File: v3/plugins/ms-vscode/vscode-github-pullrequest/0.20.0/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+publisher: ms-vscode
+apiVersion: v2
+name: vscode-github-pullrequest
+version: 0.20.0
+type: VS Code extension
+displayName: GitHub Pull Requests and issues
+title: Pull Request Provider for GitHub.
+description: This extension allows you to review and manage GitHub pull requests and issues.
+icon: https://raw.githubusercontent.com/microsoft/vscode-pull-request-github/0.20.0/resources/icons/github_logo.png
+repository: https://github.com/microsoft/vscode-pull-request-github
```
Comment (ericwill) [inline v3/plugins/ms-vscode/vscode-github-pullrequest/0.20.0/meta.yaml:10]: Publish date should match the date (roughly) that the plugin was published in our registry

Comment (vinokurig) [reply v3/plugins/ms-vscode/vscode-github-pullrequest/0.20.0/meta.yaml:10]: Fixed

**Review by ericwill — APPROVED** [review]

**Comment by vinokurig** [comment]
[ci-test]

**Comment by benoitf** [comment]
it is not depending on https://github.com/eclipse/che-theia/pull/841 ?

**Comment by ericwill** [comment]
> it is not depending on [eclipse/che-theia#841](https://github.com/eclipse/che-theia/pull/841) ?

Sorry thought this one was merged already

**Comment by ericwill** [comment]
Actually I think we have a circular dependency as https://github.com/eclipse/che-theia/pull/841#issuecomment-692520427 suggests that PR depends on this update :laughing:

**Pull request merged**